openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/EditTestCaseModal.tsx

```diff
@@ -103,10 +129,11 @@ const EditTestCaseModal: React.FC<EditTestCaseModalProps> = ({
             : value,
       })
     );
+    const description = markdownRef.current?.getEditorContent();
```

Comment:
@ShaileshParmar11, I would recommend not to use the ref for getting the description, instead get the value from the form directly.